src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -864,13 +867,52 @@ trait BronzeTransforms extends SparkSessionWrapper {
     }
   }
 
+
+  private[overwatch] def getAllEventLogPrefix(inputDataframe: DataFrame, apiEnv: ApiEnv): DataFrame = {
+    val mountMap = getMountPointMapping(apiEnv) //Getting the mount info from api and cleaning the data
+      .withColumn("source", when('source.endsWith("/"), 'source.substr(lit(0), length('source) - 1)).otherwise('source))
+      .filter(col("mount_point") =!= "/")
+
+    //Cleaning the data for cluster log path
+    val formattedInputDf = inputDataframe.withColumn("cluster_log_conf", when('cluster_log_conf.endsWith("/"), 'cluster_log_conf.substr(lit(0), length('cluster_log_conf) - 1)).otherwise('cluster_log_conf))
+      .withColumn("cluster_mount_point_temp", regexp_replace('cluster_log_conf, "dbfs:", ""))
+      .withColumn("cluster_mount_point", regexp_replace('cluster_mount_point_temp, "//", "/"))
+
+
+    //Joining the cluster log data with mount point data
+    val joinDF = formattedInputDf
+      .join(mountMap, formattedInputDf.col("cluster_mount_point").startsWith(mountMap.col("mount_point")), "left") //starts with then when
+
+    val clusterMountPointAr = split('cluster_mount_point, "/")
+    val mountPointAr = split('mount_point, "/")
+    val hasSubFolders = size(clusterMountPointAr) > size(mountPointAr)
+    val buildSubfolderSources = concat_ws("/", 'source, array_join(array_except(split('cluster_mount_point, "/"), split('mount_point, "/")), "/"))
+
+    //Generating the final source path for mount points
+    val pathsDF = joinDF.withColumn("source_temp", when(hasSubFolders, buildSubfolderSources) otherwise ('source))
+      .withColumn("derivedSource", when('source.isNull, 'cluster_mount_point) otherwise ('source_temp))
+      .withColumn("topLevelTargets", array(col("derivedSource"), col("cluster_id"), lit("eventlog")))
+      .withColumn("wildPrefix", concat_ws("/", 'topLevelTargets))
+
+    val result = pathsDF.select('wildPrefix, 'cluster_id).distinct()
```

Comment:
the `.distinct` should be after the .select('wildPrefix) in the calling function not here. Please move the .distinct()

cc @Sriram-databricks -- urgent required fix